Assets/MRTK/Tools/BuildWindow/BuildDeployWindow.cs

```diff
@@ -563,6 +565,22 @@ private void RenderAppxBuildView()
                     // since there's a null texture issue when Unity reloads the assets during a build
                     MixedRealityBuildPreferences.DrawAppLauncherModelField(appxCancellationTokenSource == null);
 
+                    // Draw the section for nuget path
+                    EditorGUILayout.LabelField("Nuget Path (Optional)", EditorStyles.boldLabel);
+
+                    string nugetExecutablePath = EditorGUILayout.TextField(NugetPathLabel, UwpBuildDeployPreferences.NugetExecutablePath);
+                    using (new EditorGUILayout.HorizontalScope())
+                    {
+                        if (GUILayout.Button("Select Nuget Executable Path"))
```

Comment:
Does this need to be exposed via the command line as well?

Reply:
Just did!